templates/pages/assets/customfield.html.twig

```diff
@@ -0,0 +1,152 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2024 Teclib' and contributors.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{% extends "generic_show_form.html.twig" %}
+{% import 'components/form/fields_macros.html.twig' as fields %}
+{% import 'components/alerts_macros.html.twig' as alerts %}
+
+{% block form_fields %}
+    <input type="hidden" name="{{ 'Glpi\\Asset\\AssetDefinition'|itemtype_foreign_key }}" value="{{ assetdefinitions_id }}">
+
+    {{ fields.textField('label', item.fields['label'], __('Label')) }}
+    <input type="hidden" name="name" value="{{ item.fields['name'] }}">
+    {{ fields.htmlField('', item.fields['name'], __('System name'), {
+        add_field_class: 'displayed-name'
+    }) }}
+    {{ fields.dropdownArrayField('type', item.fields['type']|default('string'), field_types, _n('Type', 'Types', 1)) }}
+    <div class="placeholder-type-info">
+        {{ alerts.alert_info(__('Placeholders are empty fields without a label or value to help with the form layout')) }}
+    </div>
+    {{ fields.nullField() }}
+
+    <div class="field-type-options d-flex flex-wrap"></div>
+
+    {{ fields.dropdownArrayField('itemtype', item.fields['itemtype'], allowed_dropdown_itemtypes, __('Item type'), {
+        add_field_class: 'd-none'
+    }) }}
+    {{ fields.dropdownArrayField('field_options[values]', '', item.fields['field_options']['values']|default([]), __('Allowed values'), {
+        add_field_class: 'd-none',
+        multiple: true,
+        enable_tags: true,
+        values: item.fields['field_options']['values']|default([])
+    }) }}
```

Comment:
This field is never displayed. The related code should be removed.

Handling the "allowed values" for dropdowns is an advanced feature that we did not planned yet.

```suggestion
```